image/image_linux.go

```diff
@@ -842,6 +842,10 @@ func (s *imageSeeder) warnOnUnassertedSnaps() error {
 }
 
 func (s *imageSeeder) finish() error {
+	if err := s.w.VerifySnapBootstrapCompatibility(); err != nil {
+		fmt.Fprintf(Stderr, "WARNING: %v\n", err)
```

Comment:
Is emitting a warning enough? should it be more strict and just?

cc: @pedronis @alfonsosanchezbeato

Reply:
I think this is a good suggestion, I added this in